src/widgets/ProgressDialog.cpp

```diff
@@ -1413,6 +1434,7 @@ ProgressResult ProgressDialog::Update(
    // Nyquist effects call Update on every callback, but YieldFor is
    // quite slow on Linux / Mac, so don't call too frequently. (bug 1575)
    if ((now - mYieldTimer > 50) || (value == 1000)) {
+      mYieldsCount++;
```

Comment:
Perhaps the total duration of yield time, within polling time, could be accumulated too?

Reply:
I think it will match precisely the Poll time, but this can be easily done (and might be quite educative!)